crates/sound/src/virtio_sound.rs

```diff
@@ -92,6 +92,12 @@ pub const VIRTIO_SND_PCM_FMT_DSD_U16: u8 = 22;
 pub const VIRTIO_SND_PCM_FMT_DSD_U32: u8 = 23;
 pub const VIRTIO_SND_PCM_FMT_IEC958_SUBFRAME: u8 = 24;
 
+
+pub const SUPPORTED_FORMATS: u64 = 1 << VIRTIO_SND_PCM_FMT_U8
```

Comment:
I would move this out of `virtio_sound.rs` where we should put only the stuff defined into the virtio-spec

Reply:
do you have any preference on where to put it?

Reply:
In `vhu_sound.rs` should be fine for now